### PR TITLE
Changed javadocs on methods in the card section, as well as added ske…

### PR DESCRIPTION
**Pull request by roym5**
JavaDoc changes for card as well as a small method that can be expanded for showing the number of holes

### CODE REVIEW

File: src/csc439teamplatypus/cardgame/golfgame/CLIView.java [hunk 1]
```diff
@@ -217,6 +217,18 @@ public void startGame() {
         System.out.print("Please enter a number between 2 and 7: ");
         int numOfPlayers = Integer.parseInt(input.next());
         setNumOfPlayers(numOfPlayers);
+        System.out.println("How many holes would you like to play?");
+        System.out.println("Enter 9 or 18: ");
+        int numOfHoles=Integer.parseInt(input.next());
```
Comment (nflanagan6) [inline src/csc439teamplatypus/cardgame/golfgame/CLIView.java:222]: Do we want to do Integer.parseInt(input.next()) or input.nextInt()? I don't know if it really makes any difference and I don't have a preference, but we should probably stay consistent on this

Comment (nflanagan6) [inline src/csc439teamplatypus/cardgame/golfgame/CLIView.java:222]: Do we want to do Integer.parseInt(input.next()) or input.nextInt()? I don't know if it really makes any difference and I don't have a preference, but we should probably stay consistent on this

File: src/csc439teamplatypus/cardgame/golfgame/CLIView.java [hunk 2]
```diff
@@ -217,6 +217,18 @@ public void startGame() {
...
+        howManyHoles(numOfHoles);
+    }
+
+    /**
+     *Takes numOfHoles and sets game length
+     * @param numOfHoles Number of holes being played
+     */
+    public void howManyHoles(int numOfHoles){
+        System.out.print("You are playing "+ numOfHoles +" holes of golf!");
```
Comment (nflanagan6) [inline src/csc439teamplatypus/cardgame/golfgame/CLIView.java:231]: @Hunt-E @abbiland @roym5 Do you all think we should add a \n to this or use println()? I think we ended up going back and adding line breaks a lot last lab

File: src/csc439teamplatypus/cardgame/golfgame/CLIView.java (hunk 1 above)
Comment (roym5) [reply src/csc439teamplatypus/cardgame/golfgame/CLIView.java:222]: Yeah that makes more sense most of the input would be an int anyways.

File: src/csc439teamplatypus/cardgame/golfgame/CLIView.java (hunk 2 above)
Comment (roym5) [reply src/csc439teamplatypus/cardgame/golfgame/CLIView.java:231]: Yeah that makes more sense, this code can be ignored too and I can go ahead and push the code without this additional method, I just worked on this as kind of a jumpstart before being assigned tasks. My main push was the Javadoc's before each methods let me know and I can push just that.

Comment (nflanagan6) [reply src/csc439teamplatypus/cardgame/golfgame/CLIView.java:231]: Ok, thanks @roym5. Do you mind repushing only the changes we're going to keep? 

Thanks!

**Pull request closed without merging**